compiler/luci-interpreter/src/kernels/Cast.test.cpp

```diff
@@ -44,32 +44,163 @@ void Check(std::initializer_list<int32_t> shape, std::initializer_list<T1> input
   EXPECT_THAT(extractTensorShape(output_tensor), shape);
 }
 
+template <typename T>
```

Comment:
This helper is needed because bool tensor stores data as `uint8_t` instead of `bool`.
General helper function defined above fail to compile.